src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderAlgorithmPicked.cpp

```diff
@@ -854,28 +840,27 @@ namespace OpenMS
       // used in any feature with higher intensity, we can add it to the
       // features_ list.
       std::vector<Size> seeds_contained;
-      for (std::map<Size, Feature>::iterator iter = tmp_feature_map.begin(); iter != tmp_feature_map.end(); ++iter)
+      for (auto & iter : tmp_feature_map)
```

Comment:
`iter` is not an iterator?
use `f` if its a feature

Reply:
Done